### DISCUSSION TITLE
Hi, why my cuda/thrust program can only run in some types of NVIDIA gpu cards?

### DISCUSSION THREAD

**Discussion by lfwin** [NVIDIA/cccl#1237] [Thrust]
Hi, i have a cuda12.3/thrust program which have new feature of thrust, such as thrust::universal_vector,  right now my program only pass tests on 1080ti and 3070 cards, but fails on 2060 super and 4060 notebook cards.  Why is this happened? How to solve this? Thanks in advance.

**Comment by elstehle** [comment]
Thanks for reaching out, @lfwin! To help us pinpoint the issue, could you provide a minimal code example that reproduces the problem? This will greatly assist our investigation.
